frontend/src/lib/doc-util.tsx

```diff
@@ -7,7 +7,7 @@ import { ExternalLinkAltIcon } from '@patternfly/react-icons'
 export const DOC_VERSION = '2.5'
 
 export const DOC_HOME = `https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/${DOC_VERSION}`
-export const DOC_BASE_PATH = `${DOC_HOME}/html`
+export const DOC_BASE_PATH = `${DOC_HOME}/html-single`
```

Comment:
@Randy424 This change would affect every documentation link. Is it correct? In the document @cadawson provided, it looks to me like `html-single` is used for the **Staged** docs, but `html` for the **Live** docs

Reply:
During our walkthrough of the links, @cadawson suggested changing all the ACM doc references to use html-single, as it would make it easier to navigate after landing at the doc page. We did no go back and retroactively change the reference links though.

@cadawson please correct me if I'm missing a caveat or more here.